service/src/main/java/bio/terra/catalog/rawls/RawlsService.java

```diff
@@ -14,21 +22,79 @@
 @Component
 public class RawlsService {
   private static final Logger logger = LoggerFactory.getLogger(RawlsService.class);
+  public static final List<String> ACCESS_LEVEL_LIST = List.of("accessLevel");
+  public static final List<String> ACCESS_LEVEL_AND_ID_LIST =
```

Comment:
I actually went back and forth on this. To be honest, it's mostly a habit from working in untyped languages, to make the intended type clear through the variable name. Since Java is typed, removing them feels fine, since the typing covers that info for us.